contributing.md

```diff
@@ -172,6 +175,18 @@ To get a production build, run:
 npm run build:js
 ```
 
+### Updating Google fonts 
+
+Before updating Google Fonts, an `.env` is required to add the Google Fonts API key. 
+Details of how to get an API key can be found on in the [Google fonts docs](https://developers.google.com/fonts/docs/developer_api). 
+To add an API. 
+
+1. Copy `.env.example` to `.env`. 
```

Comment:
```suggestion
1. Copy `example.env` to `.env`. 
```

Reply:
FIxed in https://github.com/ampproject/amp-wp/pull/3110/commits/1c057e221ad70b2f4c3cf421f7faee74a000331a